Makefile

```diff
@@ -164,7 +188,7 @@ generate-dnsmasq-site-policy-section.sh:
 dnsmasq-workaround: SEED_CLUSTER_NAME ?= $(SEED_VM_NAME).redhat.com
 dnsmasq-workaround: CLUSTER=$(SEED_VM_NAME)
 dnsmasq-workaround: generate-dnsmasq-site-policy-section.sh
-	./generate-dnsmasq-site-policy-section.sh --name $(SEED_VM_NAME) --domain $(CLUSTER_DOMAIN) --ip $(SEED_VM_IP) --mc | $(oc) apply -f -
+	./generate-dnsmasq-site-policy-section.sh --name $(SEED_VM_NAME) --domain $(SEED_DOMAIN) --ip $(SEED_VM_IP) --mc | $(oc) apply -f -
```

Comment:
I don't think we need it, everything was pushed already

Reply:
Great!
Removed! :)